### PR TITLE
Fix FireFox driver, minor driver extracting refactor

### PR DESCRIPTION
**Pull request by pwitusik**

### CODE REVIEW

File: G1ANT.Addon.Selenium/Api/SeleniumDriverResourceDescription.cs [hunk 1]
```diff
@@ -0,0 +1,84 @@
...
+                }
+            }
+        }
+
+        private bool DoesFileExist(string folder, string fileName)
+        {
+            return File.Exists(Path.Combine(folder, fileName));
+        }
+
+        private bool AreFilesOfTheSameLength(int length, string folder, string fileName)
```
Comment (ChrisPrusik) [inline G1ANT.Addon.Selenium/Api/SeleniumDriverResourceDescription.cs:79]: Is this enough? Maybe AreFilesOfTheSameLengthAndVersion()?

File: G1ANT.Addon.Selenium/SeleniumAddon.cs [hunk 2]
```diff
@@ -28,79 +25,32 @@ public class SeleniumAddon : Language.Addon
...
                 try
                 {
-                    proc.Kill();
+                    driver.UnpackIfNeeded(Assembly, UnpackFolder);
                 }
-                catch
+                catch (Exception ex)
                 {
-
+                    RobotMessageBox.Show(ex.Message);
```
Comment (ChrisPrusik) [inline G1ANT.Addon.Selenium/SeleniumAddon.cs:51]: Maybe we should write that information to the log?

**Review by ChrisPrusik — APPROVED** [review]

File: G1ANT.Addon.Selenium/Api/SeleniumDriverResourceDescription.cs (hunk 1 above)
Comment (pwitusik) [reply G1ANT.Addon.Selenium/Api/SeleniumDriverResourceDescription.cs:79]: This implementation has been moved from SeleniumAddon, I don't want to change the code that works fine.

File: G1ANT.Addon.Selenium/SeleniumAddon.cs (hunk 2 above)
Comment (pwitusik) [reply G1ANT.Addon.Selenium/SeleniumAddon.cs:51]: This implementation has been moved from SeleniumAddon, I don't want to change the code that works fine.

Comment (pwitusik) [reply G1ANT.Addon.Selenium/SeleniumAddon.cs:51]: I don't want to change the behaviour that was already implemented.

**Pull request merged**